Find a functionally equivalent code:com.android.internal.os.RuntimeInit.wrapperInit:COMMENT
Method Modifier: public      static      internal    
Comment:/**
 * The main function called when an application is started through a
 * wrapper process.
 *
 * When the wrapper starts, the runtime starts {@link RuntimeInit#main}
 * which calls {@link WrapperInit#main} which then calls this method.
 * So we don't need to call commonInit() here.
 *
 * @param targetSdkVersion target SDK version
 * @param argv arg strings
 */

@@ -1,5 +1,5 @@
 {
     if (DEBUG)
         Slog.d(TAG, "RuntimeInit: Starting application from wrapper");
-    applicationInit(targetSdkVersion, argv);
+    applicationInit(targetSdkVersion, argv, null);
 }

